draft-zip.md

```diff
@@ -0,0 +1,97 @@
+```
+ZIP: 
+Title: Deposit 50% of transaction fees to the Zcash Sustainability Fund
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits: Nathan Wilcox
+         Mark Henderson
+         Jason McGee
+         Tomek Piotrowski
+         Mariusz Pilarek
+Status: Draft
+Category: Ecosystem
+Created: 2023-09-
+License: BSD-2-Clause
+```
+
+# Terminology
+
+The key words “MUST”, “SHOULD”, “SHOULD NOT”, “MAY”, “RECOMMENDED”, “OPTIONAL”, and “REQUIRED” in this document are to be interpreted as described in RFC 2119. [1]
+
+The term “network upgrade” in this document is to be interpreted as described in ZIP 200. [2]
+
+The term “Block Rewards” refers to the algorithmic issuance of ZEC to every block’s creator – part of the consensus rules.
+
+“Issuance” - The method by which unmined or unissued ZEC is converted to ZEC available to users of the network
+
+“We” - the ZIP authors, owners listed in the above front matter
+
+“`ZSF_BALANCE_AFTER[h]`” is balance of the Zcash Sustainability Fund as defined in ZIP ###
+
+# Abstract
+
+This ZIP proposes a modification to transaction fees, diverting 50% of transaction fees back into the `ZSF_BALANCE_AFTER[h]`, while  the destination of the remaining 50% is unchanged and goes to the block miner,. This proposal effectively “unmints” a portion of transaction fees, contributing to a deflationary effect and offering long-term support for the Zcash network.
```

Comment:
Please see my question about justifying the 50/50 split using an incentive security model:
https://forum.zcashcommunity.com/t/the-zcash-posterity-fund/42703/104

Just to be clear, this specific suggestion is optional, but I'd like to see some kind of security argument in the ZIP. (The abstract talks about increasing security, but the rest of the ZIP doesn't explain how the proposed changes provide increased security.)